src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import 'variables';
+@import 'card';
+@import 'stars';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $font-family;
```

Comment:
The variable `$font-family` is used here, but it's not defined within this file. Ensure that it is defined in the `variables` file that is imported at the beginning.